week4:JSweek2/5.js

```diff
@@ -0,0 +1,5 @@
+var z = '7.25';
+console.log(z);
+var a = Math.ceil(7.25);
+console.log(a);
+console.log(Math.max(z, a ));
```

Comment:
nit: you have an extra space in the line, you can remove it to make the code neater!